### DISCUSSION TITLE
Is it intended that refetch is not affected by enabled?

### DISCUSSION THREAD

**Discussion by jjunyjjuny** [TanStack/query#7907] [Q&A] [answered]
here is reproduction

https://playcode.io/1973194

I'm using v4.36.1 and even if I refetch, if enabled is false, I expected fetching would not be done

However, regardless of enabled, fetch was done.

Do I need to add enabled conditions directly when calling refetch?

[this issue](https://github.com/TanStack/query/issues/1965) says that this problum has been resolved since v4, but the problem still occurs

**Comment by TkDodo** [comment, marked as answer]
`refetch` returned from `useQuery` bypasses `enabled`. That is documented here:

https://tanstack.com/query/v5/docs/framework/react/guides/disabling-queries

> refetch returned from useQuery can be used to manually trigger the query to fetch

**Reply by jjunyjjuny** [reply]
👍 👍 👍 👍 👍 

I've been misunderstanding refetch. Thank you
